socorro/external/legacy_es/super_search_fields.py

```diff
@@ -1060,6 +1060,9 @@ def apply_schema_properties(fields, schema):
         "query_type": "enum",
         "storage_mapping": {"type": "string"},
     },
+    "crash_inconsistencies": keyword_field(
+        "crash_inconsistencies",
+    ),
```

Comment:
I can't seem to search this field. In dev I process the crashes from the bug, then go to http://localhost:8000/search/?crash_inconsistencies=!__null__ I get no results, but I can see the field is populated via the details pane on individual crashes.